code/modules/antagonists/eldritch_cult/eldritch_structures.dm

```diff
@@ -0,0 +1,171 @@
+		//Both organs and bodyparts hold information if they are organic or robotic in the exact same way.
+		var/obj/item/bodypart/forced = I
+		if(forced.status != BODYPART_ORGANIC)
+			return
+
+		if(current_mass >= max_mass)
+			to_chat(user,"<span class='notice'> Crucible is already full!</span>")
+			return
+		playsound(src, 'sound/items/eatfood.ogg', 100, TRUE)
+		to_chat(user,"<span class='notice'>Crucible devours [I.name] and fills itself with a little bit of liquid!</span>")
+		current_mass++
+		qdel(I)
+		update_icon_state()
+		return
+
+	return ..()
+
+/obj/structure/eldritch_crucible/attack_hand(mob/user)
+	if(!IS_HERETIC(user) && !IS_HERETIC_MONSTER(user))
+		if(iscarbon(user))
+			devour(user)
+		return
+
+	if(in_use)
+		to_chat(user,"<span class='notice'>Crucible is already in use!</span>")
+		return
+
+	if(current_mass < max_mass)
+		to_chat(user,"<span class='notice'>Crucible isn't full! Bring it more organs or bodyparts!</span>")
+		return
+
+	in_use = TRUE
+	var/list/lst = list()
+	for(var/X in subtypesof(/obj/item/eldritch_potion))
+		var/obj/item/eldritch_potion/potion = X
+		lst[initial(potion.name)] = potion
+	var/type = lst[input(user,"Choose your brew","Brew") in lst]
+	playsound(src, 'sound/misc/desecration-02.ogg', 75, TRUE)
+	new type(drop_location())
+	current_mass = 0
+	in_use = FALSE
+	update_icon_state()
+
+/obj/structure/eldritch_crucible/proc/devour(mob/living/carbon/user)
+	if(HAS_TRAIT(user,TRAIT_NODISMEMBER))
+		return
+	playsound(src, 'sound/items/eatfood.ogg', 100, TRUE)
+	to_chat(user,"<span class='danger'>Crucible grabs your arm and devours it whole!</span>")
+	var/obj/item/bodypart/arm = user.get_active_hand()
+	arm.dismember()
+	qdel(arm)
+	current_mass += current_mass < max_mass ? 1 : 0
+	update_icon_state()
+
+/obj/structure/eldritch_crucible/update_icon_state()
+	. = ..()
+	if(current_mass == max_mass)
+		icon_state = "crucible"
+	else
+		icon_state = "crucible_empty"
+
+/obj/structure/trap/eldritch
+	name = "elder carving"
+	desc = "Collection of unknown symbols, they remind you of days long gone..."
+	icon = 'icons/obj/eldritch.dmi'
+	charges = 1
+	var/mob/owner
+
+/obj/structure/trap/eldritch/Crossed(atom/movable/AM)
+	if(!isliving(AM))
+		return ..()
+	var/mob/living/living_mob = AM
+	if((owner && living_mob == owner) || IS_HERETIC(living_mob) || IS_HERETIC_MONSTER(living_mob))
+		return
+	return ..()
+
+/obj/structure/trap/eldritch/attacked_by(obj/item/I, mob/living/user)
+	. = ..()
+	if(istype(I,/obj/item/melee/rune_knife) || istype(I,/obj/item/nullrod))
+		qdel(src)
+
+/obj/structure/trap/eldritch/proc/set_owner(mob/_owner)
+	owner = _owner
+
+/obj/structure/trap/eldritch/alert
+	name = "alert carving"
+	icon_state = "alert_rune"
+	alpha = 10
+
+/obj/structure/trap/eldritch/alert/trap_effect(mob/living/L)
+	if(owner)
+		to_chat(owner,"<span class='big boldwarning'>[L.real_name] has stepped foot on the alert rune in [get_area(src)]!</span>")
+	return ..()
```

Comment:
imo, this alert effect really should just be rolled into the effects of the other traps, with this third trap being... something else. because of the restriction of not being able to place traps next to each other, I feel like this doesn't synergize with the other traps as much as it should.

Reply:
maybe the replacement for this rune would be a rune that gives you a prompt to teleport to it when it's triggered? sort of like the hag's runes from dead by daylight.

Reply:
I dont really want to make the runes do more than they already do, alert rune has a very specific purpose and i dont want to mix it with other ones.